docs/pages/core-concepts.mdx

```diff
@@ -46,61 +47,87 @@ Services**, which can run in private networks. This means that, in the Proxy
 Service's minimal configuration, you can expose only port `443` to the internet
 and run the rest of your infrastructure in private networks.
 
-You can also configure clients to bypass Proxy Service instances and connect to
-resources with Teleport-issued certificates directly.
+As a key component of the Teleport control plane, the Proxy Service handles secure routing and session recording 
+for all incoming connections. For Teleport Cloud users, the Proxy Service is fully managed and hosted by Teleport.
 
-Read our guide to [how the Teleport Proxy Service
-works](reference/architecture/proxy.mdx).
+Read our guide to [how the Teleport Proxy Service works](reference/architecture/proxy.mdx).
 
 ## Teleport services
 
 A **Teleport service** manages access to resources in your infrastructure, such
-as Kubernetes clusters, Windows desktops, internal web applications, and
-databases.
+as Kubernetes clusters, Windows desktops, internal web applications, etc.
 
-A single running `teleport` process can run one or more **Teleport services**,
-depending on the user's configuration. Read about all subcommands of `teleport`
-in our [CLI Reference](./reference/cli/teleport.mdx).
+Each Teleport process can run one or more services. Different services enable different Teleport functions. For example, the `ssh_service` provides SSH access
```

Comment:
Would it make sense to use the service names we introduce in this guide instead of the names of config fields? I would use "SSH Service" instead of `ssh_service` and "Database Service" instead of `database_service`.